packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -52,6 +73,10 @@ const awaitVisitor = {
 };
 
 export default function(path: NodePath, file: Object, helpers: Object) {
+  path.traverse(yieldVisitor, {
+    visited: new WeakSet(),
+  });
+
   path.traverse(awaitVisitor, {
```

Comment:
Can you merge `yieldVisitor` and `awaitVisitor`, to avoid traversing the function body twice?